misc/msvs/vcxproj.template

```diff
@@ -10,16 +10,16 @@
     <VCProjectUpgraderObjectName>NoUpgrade</VCProjectUpgraderObjectName>
   </PropertyGroup>
   %%PROPERTIES%%
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" Condition=" exists('$(VCTargetsPath)\Microsoft.Cpp.Default.props') "/>
   <PropertyGroup Label="Configuration">
     <ConfigurationType>Makefile</ConfigurationType>
     <UseOfMfc>false</UseOfMfc>
-    <PlatformToolset>v143</PlatformToolset>
+    <DefaultPlatformToolset Condition="'$(DefaultPlatformToolset)'==''"/>
```

Comment:
This doesn't do anything, what are you trying to do here?

Reply:
DefaultPlatformToolset should be defined at least to an empty value, otherwise Rider is not happy. Start suggesting to switch MSBuild to a different version.

Reply:
Specifically Rider has inspection against the case, when DefaultPlatformToolset is not set at all.
Empty DefaultPlatformToolset indicates that there are no Cpp props and targets.
However Godot works perfectly fine with any msbuild without those props.
VS doesn't mind `<DefaultPlatformToolset Condition="'$(DefaultPlatformToolset)'==''"/>` because it sets its own value there.

Reply:
We will fix that in Rider https://youtrack.jetbrains.com/issue/RIDER-123783/Improve-DefaultPlatformToolsetIsMissingDiagnostic in 2025.2 (hopefully)